import grpc
import mine_grpc_pb2_grpc
import mine_grpc_pb2

def run():



class Services():
    def getTransactionId(self, request, context):
        returns (intResult)

    def getChallenge(self, request, context):
        returns (intResult)

    def getTransactionStatus(self, request, context):
        returns (intResult)

    def submitChallenge(self, request, context):
        returns (intResult)

    def getWinner(self, request, context):
        returns (intResult)

    def getSolution(self, request, context):
        returns (structResult)

if __name__ == '__main__':
    run()